ruby-challenges.rb

```diff
@@ -10,23 +17,219 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def beverages_array (letters)
+    beverages_array = ['coffee', 'tea', 'juice', 'water', 'soda water']
+    letter_o = 'o'
+    letter_t = 't'
+    if letters.include?(letter_o)
+        ['coffee', 'soda water']
+    
+    elsif letters.include?(letter_t)
```

Comment:
Similar to a function, we want to create dynamic code with methods. I think you should give this one another attempt.